spec/load_fixtures_helper.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper.rb'
+
+RSpec.configure do |rspec|
+  # This config option will be enabled by default on RSpec 4,
+  # but for reasons of backwards compatibility, you have to
+  # set it on RSpec 3.
+  #
+  # It causes the host group and examples to inherit metadata
+  # from the shared context.
+  rspec.shared_context_metadata_behavior = :apply_to_host_groups
+end
+
+RSpec.shared_context "fixture moabs in db" do
+  before(:all) do
+    setup
+    load_fixture_moabs
+  end
+  after(:all) do
+    # TODO: danger - if there are objects in the test db we want to keep
+    Settings.moab.storage_roots.each do |_name, storage_root|
```

Comment:
or actually, since `setup` has been called, could you just use `@moab_storage_dirs` here instead of looping over `Settings.moab.storage_roots` again?

Reply:
no, because the instance vars aren't available to before/after(:all) blocks

Reply:
ah, right, sorry